src/xdocs/config_coding.xml

```diff
@@ -36,16 +36,21 @@ int[] a = new int[]
         </source>
 
         <p>
-          The check allows leaving out the comma at the end if both the left and right curly brackets
-          are on the same line.
+          The check allows leaving out the comma at the end if either left
+          or right curly braces are on the same line as the last element of the array.
         </p>
         <source>
 return new int[] { 0 };
+return new int[] { 0
+        };
+return new int[] {
+        0 };
         </source>
 
         <p>
           Rationale: Putting this comma in makes it easier to change the order
-          of the elements or add new elements on the end.
+          of the elements or add new elements on the end. Main benefit of a trailing
+          comma is that when you add new entry to an array, no surrounding lines are changed.
```

Comment:
@romani I assume you mean the examples in this post: https://github.com/checkstyle/checkstyle/issues/1509#issuecomment-138005191
I think it is more than just the code examples, the comments around it explaining the build up of the change history also helps. That's the post that makes the most sense when I was first reading this issue.
I think we should plagiarize mkordas and describe the rationale in steps like he does.